src/jgo/jgo.py

```diff
@@ -271,7 +271,7 @@ def run_and_combine_outputs(command, *args):
 
 def find_endpoint(argv, shortcuts={}):
     # endpoint is first positional argument
-    pattern = re.compile("(.*https?://.*|[a-zA-Z]:\\.*)")
+    pattern = re.compile("(.*(https?|grpc)://.*|[a-zA-Z]:\\.*)")
```

Comment:
IMO this just kicks the can down the road.  Another protocol will hit this problem, and be needed to be added to the regex.

For the case mentioned in https://github.com/scijava/jgo/issues/92 , the problem is that the `--` isnt being treat with sufficient priority.  Everything after the `--` should be ignored.